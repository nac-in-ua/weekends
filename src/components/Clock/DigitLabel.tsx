import React from 'react'

type PropsType = {
  label: string
}

const DigitLabel = ({ label }: PropsType) => {
  console.log('label remdered!')

  return (
    <div className="flex h-8 items-center justify-center text-lg font-thin uppercase">
      {label}
    </div>
  )
}

export default React.memo(DigitLabel)
