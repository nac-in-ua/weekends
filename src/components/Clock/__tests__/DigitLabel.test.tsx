import DigitLabel from '../DigitLabel'
import { render, screen } from '@testing-library/react'

describe('DigitLabel', () => {
  it('should render label', () => {
    const { container } = render(<DigitLabel label="my label" />)
    const label = screen.getByText(/my label/i)
    expect(label).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})
