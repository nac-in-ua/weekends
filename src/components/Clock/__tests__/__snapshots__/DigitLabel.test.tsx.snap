// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DigitLabel should render label 1`] = `
<div>
  <div
    class="flex h-8 items-center justify-center text-lg font-thin uppercase"
  >
    my label
  </div>
</div>
`;
